README.md

```diff
@@ -3,13 +3,13 @@
 Exercism exercises in Objective-C
 
 ## Try it!
-If you've downloaded the [command-line client](http://exercism.io/clients/cli) and have Objective-C installed on your machine, then go ahead and fetch the first problem.
+If you've downloaded the [command-line client](http://exercism.org/clients/cli) and have Objective-C installed on your machine, then go ahead and fetch the first problem.
```

Comment:
This link gives a 404, can you please find the equivalent and use that link instead?